dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements.txt
 
+pytest==4.2.0
```

Comment:
Even though `pytest` is a dependency of `pytest-cov`, we import it explicitly in tests so it should be present in our requirements.